docs/configuration_and_usage.rst

```diff
@@ -99,6 +99,77 @@ to the help strings provided by the form.
 You can preview emails in the template edit view using the "Preview (program)" and "Preview (course)" buttons in
 top-right corner.
 
+Integrated Channels
+-------------------
+
+SAP SuccessFactors Configuration
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+One of the integrated channels available for an ``EnterpriseCustomer`` is `SAP SuccessFactors`_.
+
+To integrate an ``EnterpriseCustomer`` with SAP SuccessFactors:
+
+* Set up a `TPA SAML Integration`_ for the SAP SuccessFactors endpoint.
+* From the LMS Admin, ensure that your ``EnterpriseCustomer`` record has its ``identity_provider`` field pointing to the
+  SAML Integration created in the previous step.
+
+  This allows learners to create accounts using Single Sign-On from SAP SuccessFactors, and to have their course
+  completion records sent back to SAP SuccessFactors.
+* From the LMS Admin, create an ``SAP SuccessFactors Enterprise Customer Configuration`` record, and link it to your
+  ``EnterpriseCustomer``.
+
+  Ensure that the record is marked ``Active`` to allow the management commands below to send data via this channel.
+
+.. _SAP SuccessFactors: https://www.successfactors.com
+.. _TPA SAML Integration: http://edx.readthedocs.io/projects/edx-installing-configuring-and-running/en/latest/configuration/tpa/tpa_SAML_SP.html
+
+Management Commands
+^^^^^^^^^^^^^^^^^^^
+
+These Django management commands send data to ``EnterpriseCustomer``'s active, integrated channels.
+
+They must be run from the command line on a server with the Open edX environment installed.
+
+.. note::
+
+   If data sharing consent is enabled for your ``EnterpriseCustomer``, then any Enterprise learners enrolled in the
+   associated courses must consent to data sharing, thus permitting their data to be sent back to SAP SuccessFactors.
+
+   If data sharing consent is *not* enabled for your ``EnterpriseCustomer``, then learner data may be sent to SAP
```

Comment:
@mattdrayer CC @griffresch  I agree that requiring explicit learner consent is surely safer, so we should exclude from this transmitted data any EnterpriseCustomers that don't require consent, and remove this warning note.  It does make me wonder about the use case for EnterpriseCustomers who don't require data sharing consent -- but I don't know the full Enterprise project scope.

In the interests of keeping this PR on task with ENT-259, can we create a separate ticket for fixing this issue?  Note that https://github.com/edx/edx-enterprise/pull/63 has the same concern, but since it shares code with https://github.com/edx/edx-enterprise/pull/59 (merged), we should be able to fix it in [one place](https://github.com/edx/edx-enterprise/blob/master/integrated_channels/integrated_channel/management/commands/__init__.py#L92-L101).  CC @haikuginger